From 5019e15587feecfa8e70638ac8bf7f8f6f8932c5 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Fri, 14 Jun 2013 11:57:00 -0700
Subject: [PATCH 126/382] MIPS: Remove dead code from Makefile.

Based on SDK octeon3_3.10.

Commit 520b3e0f5 (Add octeon2 build and configuration option) left
some dead code.  Get rid of it.

Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/Makefile |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/arch/mips/Makefile b/arch/mips/Makefile
index 4c72592..51556db 100644
--- a/arch/mips/Makefile
+++ b/arch/mips/Makefile
@@ -169,13 +169,6 @@ endif
 endif
 endif
 
-ifdef	_NOTDEF_
-cflags-$(CONFIG_CPU_CAVIUM_OCTEON) += $(call cc-option,-march=octeon) -Wa,--trap
-ifeq (,$(findstring march=octeon, $(cflags-$(CONFIG_CPU_CAVIUM_OCTEON))))
-cflags-$(CONFIG_CPU_CAVIUM_OCTEON) += -Wa,-march=octeon
-endif
-endif
-
 cflags-$(CONFIG_CPU_CAVIUM_OCTEON) += -DCVMX_BUILD_FOR_LINUX_KERNEL=1 -DUSE_RUNTIME_MODEL_CHECKS=1
 
 cflags-$(CONFIG_CAVIUM_CN63XXP1) += -Wa,-mfix-cn63xxp1
-- 
1.7.0.4

